examples/llvm/polymorphic_codata_polymorphically.effekt

```diff
@@ -0,0 +1,18 @@
+interface Eff[A] {
```

Comment:
Note that this is actually just a (minor) minification of [pos/higher_rank_polymorphism.effekt](https://github.com/effekt-lang/effekt/blob/master/examples/pos/higher_rank_polymorphism.effekt). I'm not sure whether we need this additional test sample.

Reply:
Ah, good point. I'll try if that also detects this change.

Reply:
It does, so I removed the new test case. Thanks.